prime-angular-frontend/src/app/modules/adjudication/pages/site-overview/site-overview.component.html

```diff
@@ -134,7 +134,7 @@
               Required
             </mat-error>
             <mat-error *ngIf="pec.hasError('assignable')">
-              Site ID has already been used by another site
+              Site ID has already been used by another organization
```

Comment:
Depending on screen size of browser window, validation text can overrun ... use "Site ID already used by another organization" instead?
![image](https://github.com/bcgov/moh-prime/assets/7586665/a0410a91-4a90-4a44-aa22-dd1278c0616e)

Reply:
It is updated